assets/js/base/components/product-price/index.tsx

```diff
@@ -161,7 +194,7 @@ const ProductPrice = ( {
 				regularPriceStyle={ regularPriceStyle }
 			/>
 		);
-	} else if ( minPrice !== null && maxPrice !== null ) {
+	} else if ( minPrice && maxPrice ) {
```

Comment:
This will evaluate to true if either of these values are `0` - is this going to cause problems? Can you test it and see what happens?

Reply:
A number `0` evaluates to false so this expression would be false in that case. The intent here is that if either the minPrice or maxPrice is either an empty string `''` or number `0`, this will evaluate to false and not run

Reply:
Sorry, I got my logic mixed up! But thanks for understanding what I meant and fixing it anyway :)